### DISCUSSION TITLE
Question 98: What happened to the Remarks field in esites? (JD-IZ)

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1218] [Q&A] [answered]
Description: I remember there being a field called Remarks somewhere in the sites module. I still have it in my list view, and some of our records have data in it (e.g. IRN 730107) but it does not appear to be in any of the tabs we currently have visible. Can we get the field back, or if it’s not a field we’re supposed to be using, could we get that data moved somewhere where it is visible (e.g. Notes)?

- Listview shared with Admin: JEDA-megasite view

**Comment by rondlg** [comment, marked as answer]
Response: 

@dangeloje The field you are looking for is  `LocLocalityRemarks` and it is on the Locality tab.

![image](https://github.com/user-attachments/assets/afd977df-1a7c-4f9a-b9c5-10e9d8b261e5)

**Comment by dangeloje** [comment]
Thanks. Am I supposed to have access to the Locality tab? I don't see it anywhere, but presumably I used to have access to it.

**Reply by rondlg** [reply]
@dangeloje I have created a helpdesk ticket for adding the tab to the IZMgr and IZUser group. 

- https://fieldmuseum.zendesk.com/agent/tickets/75436

**Comment by rondlg** [comment]
IZUser and IZMgr now have access to AllLocTab
